src/Auth0.php

```diff
@@ -258,6 +258,36 @@ public function login($state = null, $connection = null)
         exit;
     }
 
+    public function clientCredentials()
```

Comment:
Should we call this method something else because it essentially generates the access token

Reply:
The type of grant we are using in oauth2 terminology is called a client credentials grant, so I figured the name should reflect the action.